code/modules/uplink/uplink_items.dm

```diff
@@ -886,16 +886,16 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "Stimpacks, the tool of many great heroes, make you nearly immune to stuns and knockdowns for about \
 			5 minutes after injection."
 	item = /obj/item/reagent_containers/syringe/stimulants
-	cost = 5
+	cost = 3
 	surplus = 90
 
 /datum/uplink_item/stealthy_tools/mulligan
 	name = "Mulligan"
 	desc = "Screwed up and have security on your tail? This handy syringe will give you a completely new identity \
 			and appearance."
 	item = /obj/item/reagent_containers/syringe/mulligan
-	cost = 4
-	surplus = 30
+	cost = 1 //Legit never used
```

Comment:
Rarity of use should rarely be the sole justification for decreasing something's price. Mulligan syringes in particular are something that are so rarely used due to the very specific niche they fill

Reply:
Yep, they are only seem to be used if you want to be gotten.
Or can be shoot into someone to make them no longer protected under the AI if crewmovs on